observable_operator_test.go

```diff
@@ -607,6 +607,95 @@ func Test_Observable_IgnoreElements_Parallel_Error(t *testing.T) {
 	Assert(context.Background(), t, obs, IsEmpty(), HasError(errFoo))
 }
 
+func Test_Observable_Join(t *testing.T) {
```

Comment:
The unit test is not consistent with our way to test observables. You should use the rxgo assertion API instead.

Reply:
Fixed